### DISCUSSION TITLE
Why does SENTRY_DIST have to be a number

### DISCUSSION THREAD

**Discussion by chrismcleod** [getsentry/sentry-react-native#2394] [Q&A] [answered]
Builds were working with a dist like `android.150` now they don't bc of sentry.gradle line 77.  The docs even recommend having platform name in the dist.  https://docs.sentry.io/platforms/react-native/troubleshooting/#different-bundles-for-different-platforms

What is the recommendation now?

**Comment by marandaneto** [comment, marked as answer]
@chrismcleod https://github.com/getsentry/sentry-react-native/pull/2395

**Reply by chrismcleod** [reply]
@marandaneto thank you!

**Reply by chrismcleod** [reply]
I'm curious why the log says `versionCode: '$versionCode' isn't an Integer, but it must be.`

Why must it be an integer?  Is an integer dist better in some way?  Or is this just an opportunity for a better log message?

**Reply by marandaneto** [reply]
Because Android and iOS make versionCode and buildNumber as an Integer.
But Sentry accepts a String and we forgot that, now it's possible, we have to release the bug fix.
